tests/unit/test_encoder_error_conditions.py

```
@@ -0,0 +1,344 @@
+
+        # Create encoder
+        self.encoder = FFmpegEncoder(self.config, self.logger)
+
+    def teardown_method(self):
+        """Clean up after each test method"""
+        # Stop any running process
+        if self.encoder.process:
+            try:
+                self.encoder.process.terminate()
+            except:
+                pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)